examples/java/twitter/src/main/java/org/apache/beam/examples/twitterstreamgenerator/README.md

```diff
@@ -34,6 +34,7 @@ This directory contains an example pipelines for how to perform continues stream
 
 - Java 8
 - Twitter developer app account and streaming credentials.
+- Direct runner or Flink runner.
```

Comment:
Also Dataflow?

Reply:
No, i haven't added the support for AutoValue, my understanding is that without that the code will not run on Dataflow.